internal/satellite/module/api/module.go

```diff
@@ -27,7 +27,7 @@ var ShutdownHookTime = time.Second * 5
 
 // Module id a custom plugin interface, which defines the processing.
 type Module interface {
-
+	DependencyInjection(modules ...Module)
```

Comment:
Miss comments for core level interface.

Reply:
If you want to run DI, how could you figure out the elements of different indices, and do the typecast?